src/pages/Proposal.vue

```diff
@@ -237,6 +250,29 @@ export default defineComponent({
 		toggleModalVisible() {
 			this.isModalVisible = !this.isModalVisible;
 		},
+		//  execute_before must be after voting_end
+		async validateExecuteBefore(_rule: Rule, value: string) {
+			if (value === null) {
+				return Promise.reject("Please input execute before time.");
+			} else {
+				if (
+					this.formState.execute_before !== undefined &&
+					this.formState.vote_date !== undefined &&
+					this.formState.vote_date[1].length
```

Comment:
I think you meant `this.formState,vote_date.length`. please check

Reply:
no, its [1], to see if end date is added, [0] is start date, but I think I can update it with 0 as well